functions/pipeline/shared/db_access/db_access_v2.py

```diff
@@ -267,16 +267,15 @@ def checkout_images(self, image_count, user_id):
                     "join image_info i on i.imageid = its.imageid "
                     "join tag_state ts on ts.tagstateid = its.tagstateid "
                     "where  "
-                        "its.tagstateid in ({1},{2}) "
+                        "its.tagstateid in ({1}) "
                     "limit {0}")
-                cursor.execute(query.format(image_count, ImageTagState.READY_TO_TAG, ImageTagState.TAG_IN_PROGRESS))
```

Comment:
Missed this earlier, but don't we also want to include INCOMPLETE_TAG?

Reply:
we could include incomplete tag state, i think. @andrebriggs ?